src/stores/useMazeStore.jsx

```diff
@@ -0,0 +1,70 @@
+import { create } from "zustand";
+
+export const useMazeStore = create((set) => ({
+  username: "",
+  gameOn: false,
```

Comment:
Maybe this could be called isGameOn to follow the naming convention for booleans?